### ISSUE TITLE
skim reading allow user to configure which function keys will not interrupt say all

### ISSUE THREAD

**Issue by sine-w** [nvaccess/nvda#12000]
### Is your feature request related to a problem? Please describe.
At present, although skim reading will not be interrupted by nvda find previous and next hotkey, but it will still be interrupted by non nvda find next hotkey, e.g. 
f3 in text editor notepad++
shift + F4 in word

### Describe the solution you'd like
As nvda cannot know every search command of every programs
nvda should allow user to configure which set of hotkeys that will not interrupt say all

### Describe alternatives you've considered
nvda can  include some common search hotkeys, like f3, to be considered as not interrupting say all

**Comment by CyrilleB79** [comment]
This seems rather specific and usually NVDA does not offer such level of configuration. Anyway, I will let NVAccess team indicate themselves if they imagine such a feature in NVDA's core.
Maybe an add-on would could do the job?
I think it would make sense to integrate it in [Tony's Enhancements](https://github.com/mltony/nvda-tonys-enhancements) add-on written by @mltony  since this feature request has some similarity with Tony's dynamic keystrokes feature.

**Comment by feerrenrut** [comment]
I think @CyrilleB79 is right, to making this user configurable would be better as an add-on.

However, it would be worth clarifying if the F3 key is bound to anything in this case. I think it would make sense for non-bound keys that are passed through to the underlying application should not interrupt skim reading. That said, for shortcuts that don't move the caret, but instead show a menu or similar this might be confusing. I think it would be worth experimenting with the following different approaches:
- don't interrupt for unbound keys
- don't interrupt if we can confirm that the result of the key press is only a caret move.
